### PR TITLE
More Zeng-Hu Contractor Items

### PR DESCRIPTION
**Pull request by furrycactus**
- Added new clothing items for Zeng-Hu contractors; two berets and a labcoat."
- Fixed the mislabeling of the alternate Zeng-Hu uniform, it's now named 'zeng-hu white uniform' instead of 'zeng-hu black uniform', because it is in fact white and not black.

Also added some armour values to other contractor jumpsuits to put them more in line with their NT uniform counterparts.

### CODE REVIEW

**Comment by furrycactus** [comment]
!review

File: code/modules/clothing/under/jobs/civilian.dm [hunk 1]
```diff
@@ -166,6 +166,8 @@
 	desc = "A uniform worn by Hephaestus Industries engineers."
 	icon_state = "heph_engineer"
 	worn_state = "heph_engineer"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 10)
```
Comment (Geevies) [inline code/modules/clothing/under/jobs/civilian.dm:169]:
```suggestion
	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 10)
```

File: code/modules/clothing/under/jobs/civilian.dm [hunk 2]
```diff
@@ -178,6 +180,8 @@
 	desc = "A uniform worn by Hephaestus Industries researchers."
 	icon_state = "heph_research"
 	worn_state = "heph_research"
+	permeability_coefficient = 0.50
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 10, bio = 0, rad = 10)
```
Comment (Geevies) [inline code/modules/clothing/under/jobs/civilian.dm:184]:
```suggestion
	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 10, bio = 0, rad = 10)	
```

File: code/modules/clothing/under/jobs/civilian.dm [hunk 3]
```diff
@@ -190,6 +194,8 @@
 	desc = "A uniform worn by Necropolis Industries researchers."
 	icon_state = "necro_research"
 	worn_state = "necro_research"
+	permeability_coefficient = 0.50
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 10, bio = 0, rad = 0)
```
Comment (Geevies) [inline code/modules/clothing/under/jobs/civilian.dm:198]:
```suggestion
	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 10, bio = 0, rad = 0)	
```

File: code/modules/clothing/under/jobs/civilian.dm [hunk 4]
```diff
@@ -214,12 +220,16 @@
 	desc = "A uniform worn by Einstein Engines Incorporated employees and contractors."
 	icon_state = "einstein_engine"
 	worn_state = "einstein_engine"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 10)
```
Comment (Geevies) [inline code/modules/clothing/under/jobs/civilian.dm:223]:
```suggestion
	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 10)
```

File: code/modules/clothing/under/jobs/civilian.dm [hunk 5]
```diff
@@ -214,12 +220,16 @@
...
+	siemens_coefficient = 0.75	
 
 /obj/item/clothing/under/rank/zeng
 	name = "Zeng-Hu uniform"
 	desc = "A uniform worn by Zeng-Hu medical doctors and researchers."
 	icon_state = "zeng_research"
 	worn_state = "zeng_research"
+	permeability_coefficient = 0.50
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 10, bio = 10, rad = 0)
```
Comment (Geevies) [inline code/modules/clothing/under/jobs/civilian.dm:232]:
```suggestion
	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 10, bio = 10, rad = 0)	
```

File: code/modules/clothing/under/jobs/civilian.dm [hunk 6]
```diff
@@ -235,4 +245,6 @@
 	name = "Eridani PMC medic uniform"
 	desc = "A uniform worn by Eridani PMC paramedics."
 	icon_state = "eridani_medic"
-	worn_state = "eridani_medic"
\ No newline at end of file
+	worn_state = "eridani_medic"
+	permeability_coefficient = 0.50
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 10, rad = 0)
```
Comment (Geevies) [inline code/modules/clothing/under/jobs/civilian.dm:250]:
```suggestion
	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 10, rad = 0)		
```

**Pull request merged**